spec/board_spec.rb

```diff
@@ -1,76 +1,75 @@
+                                                      ' o | o |   ',
+                                                      '-----------',
+                                                      ' x | o |   '
+                                                    ])
     end
-    it 'Updates board for o symbol' do
-      game = Game.new
-      game.board.update([4, 5, 6], ' o ')
-      expect(game.board.display_board(game)).to eql [
-        '   |   |   ',
-        '-----------',
-        ' o | o | o ',
-        '-----------',
-        '   |   |   '
-      ]
+
+    it 'splits taken fields and updates the board after every move' do
+      game.taken_fields = [1, 4, 2, 5, 7, 8, 6, 9, 3]
+      expect(game.board.display_board(game)).to eql([
+                                                      ' x | x | x ',
+                                                      '-----------',
+                                                      ' o | o | x ',
+                                                      '-----------',
+                                                      ' x | o | o '
+                                                    ])
     end
-    it 'Updates board for o symbol in negative case' do
-      game = Game.new
-      game.board.update([4, 5, 6], ' o ')
-      expect(game.board.display_board(game)).not_to eql [
-        '   |   |   ',
-        '-----------',
-        ' o | o | x ',
-        '-----------',
-        '   |   |   '
-      ]
+
+    it 'splits taken fields and updates the board after every move' do
+      game.taken_fields = [1, 2, 3, 7, 4, 9, 6, 8]
+      expect(game.board.display_board(game)).to eql([
+                                                      ' x | o | x ',
+                                                      '-----------',
+                                                      ' x |   | x ',
+                                                      '-----------',
+                                                      ' o | o | o '
+                                                    ])
     end
-    it 'Updates board for x symbol in negative case' do
-      game = Game.new
-      game.board.update([1, 2, 3], ' x ')
-      expect(game.board.display_board(game)).not_to eql [
-        '   |   |   ',
-        '-----------',
-        ' x | x | x ',
-        '-----------',
-        '   |   |   '
-      ]
+
+    it 'splits taken fields and updates the board after every move' do
+      game.taken_fields = [1, 5, 3, 2, 8, 4, 9, 6]
+      expect(game.board.display_board(game)).to eql([
+                                                      ' x | o | x ',
+                                                      '-----------',
+                                                      ' o | o | o ',
+                                                      '-----------',
+                                                      '   | x | x '
+                                                    ])
     end
   end
```

Comment:
- [ ] Lets test for an empty board

Reply:
Implemented a test for an empty board and added several negative scenarios